feat(statistics): add business report metrics query

shanegrouber opened the pull request:
- Implement useBusinessReportMetricsQuery for fetching metrics data
- Update PortfolioRiskStatistics to utilize new metrics structure

(this code is so meticulous that even your comments deserve a round of applause)

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

## Release Notes

- **New Features**
	- Introduced a new hook for fetching business report metrics.
	- Added a new endpoint to retrieve business report metrics.
	- Implemented new data transfer objects for risk level counts and business report metrics.

- **Improvements**
	- Updated the `Statistics` component to utilize the new business report metrics.
	- Enhanced `PortfolioRiskStatistics` component to reflect new data structure and rendering logic.

- **Bug Fixes**
	- Adjusted error handling and loading state management in the `Statistics` component.

- **Chores**
	- Updated internal references and schemas to align with new data structures.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

shanegrouber committed:
feat(statistics): add business report metrics query

- Implement useBusinessReportMetricsQuery for fetching metrics data
- Update PortfolioRiskStatistics to utilize new metrics structure

(this code is so meticulous that even your comments deserve a round of applause)

MatanYadaev commented on apps/backoffice-v2/src/pages/Statistics/components/PortfolioRiskStatistics/PortfolioRiskStatistics.tsx:
```diff
@@ -44,10 +43,10 @@ export const PortfolioRiskStatistics: FunctionComponent<
         <div className={'min-h-[27.5rem] rounded-xl bg-[#F6F6F6] p-2'}>
           <Card className={'flex h-full flex-col px-3'}>
-            <CardHeader className={'pb-1'}>Portfolio Risk</CardHeader>
+            <CardHeader className={'pb-1'}>Merchant Monitoring Risk </CardHeader>
```
```suggestion
            <CardHeader className={'pb-1'}>Merchant Monitoring Risk</CardHeader>
```

shanegrouber committed:
style(PortfolioRiskStatistics): remove unnecessary space in card header

- Adjust spacing in the card header for consistent styling

(Your formatting is so inconsistent, it makes a toddler's coloring book look organized)

MatanYadaev committed:
Merge branch 'dev' of github.com:ballerine-io/ballerine into BAL-3204

MatanYadaev committed:
Merge branch 'dev' of github.com:ballerine-io/ballerine into BAL-3204

MatanYadaev committed:
Merge branch 'dev' into BAL-3204
